server/api.go

```diff
@@ -385,12 +385,12 @@ func (d dexAPI) RevokeRefresh(ctx context.Context, req *api.RevokeRefreshReq) (*
 func (d dexAPI) GetVehiclePrivilegeToken(ctx context.Context, req *api.GetVehiclePrivilegeTokenReq) (*api.GetVehiclePrivilegeTokenResp, error) {
 	expiry := d.serverConfig.Now().Add(time.Minute * 10) // TODO - Discuss an appropriate time
```

Comment:
Could we use the lifetime configured for the Dex instance? This is a bit buried.